src/ContainerRenameModal.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import {
+    Button,
+    Form, FormGroup,
+    Modal, TextInput
+} from '@patternfly/react-core';
+import cockpit from 'cockpit';
+
+import * as client from './client.js';
+
+import { ErrorNotification } from './Notification.jsx';
+import * as dockerNames from 'docker-names';
+
+const _ = cockpit.gettext;
+
+class ContainerRenameModal extends React.Component {
```

Comment:
In new code we try to use functional components and `useState()`, see [React hooks docs](https://reactjs.org/docs/hooks-state.html). If you don't feel comfortable with them, we are happy to help with this.

Reply:
Fixed in commit https://github.com/cockpit-project/cockpit-podman/pull/966/commits/0c3706e1009fabc1fee2a17b247175993469a9c4